scripts/migrateServerDB/index.ts

```diff
@@ -1,38 +1,91 @@
+// Force server mode for migration
+process.env.NEXT_PUBLIC_SERVICE_MODE = 'server';
```

Comment:
we can't force server mode  because it will always run after build, the script is here:


https://github.com/lobehub/lobe-chat/blob/main/package.json#L30

if we force server mode, it will fail on client db mode.

Reply:
@arvinxx very clear thanks very much !